sympy/physics/mechanics/inertia.py

```diff
@@ -0,0 +1,195 @@
+from sympy.core.backend import sympify
+from sympy.physics.vector import Point, Dyadic, ReferenceFrame
+
+__all__ = ['inertia', 'inertia_of_point_mass', 'Inertia']
+
+
+def inertia(frame, ixx, iyy, izz, ixy=0, iyz=0, izx=0):
```

Comment:
I wouldn't move these to a new module unless you absolutely have to. Or if you move it, does the old direct import still work?

Reply:
No, but I can try to make it work. A big plus of moving is not only that it is sensible to split up that functions file a bit, but it also removes to circular import errors, which were currently solved using `import` inside functions.

Reply:
Is it a genuine concern that a user would have been importing inertia from `sympy.physics.mechanics.functions` as opposed to just from `sympy.physics.mechanics`?

Reply:
They should not use it with a direct import, but I'll add a deprecation warning nonetheless.

Reply:
What you've done here looks good.